[2025-03-30T01:50:28.133+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-30T01:50:28.396+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ecommerce_data_pipeline.fetch_data manual__2025-03-30T01:50:25.698169+00:00 [queued]>
[2025-03-30T01:50:28.409+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ecommerce_data_pipeline.fetch_data manual__2025-03-30T01:50:25.698169+00:00 [queued]>
[2025-03-30T01:50:28.410+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-03-30T01:50:28.437+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): fetch_data> on 2025-03-30 01:50:25.698169+00:00
[2025-03-30T01:50:28.453+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1713) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-30T01:50:28.456+0000] {standard_task_runner.py:72} INFO - Started process 1715 to run task
[2025-03-30T01:50:28.456+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'ecommerce_data_pipeline', 'fetch_data', 'manual__2025-03-30T01:50:25.698169+00:00', '--job-id', '396', '--raw', '--subdir', 'DAGS_FOLDER/Gestioncommands.py', '--cfg-path', '/tmp/tmpnib8oc_y']
[2025-03-30T01:50:28.458+0000] {standard_task_runner.py:105} INFO - Job 396: Subtask fetch_data
[2025-03-30T01:50:28.539+0000] {task_command.py:467} INFO - Running <TaskInstance: ecommerce_data_pipeline.fetch_data manual__2025-03-30T01:50:25.698169+00:00 [running]> on host 778b271a0247
[2025-03-30T01:50:28.671+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='admin' AIRFLOW_CTX_DAG_ID='ecommerce_data_pipeline' AIRFLOW_CTX_TASK_ID='fetch_data' AIRFLOW_CTX_EXECUTION_DATE='2025-03-30T01:50:25.698169+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-30T01:50:25.698169+00:00'
[2025-03-30T01:50:28.674+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-03-30T01:50:28.675+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-03-30T01:50:28.676+0000] {logging_mixin.py:190} INFO - Current task name:fetch_data state:running start_date:2025-03-30 01:50:28.397737+00:00
[2025-03-30T01:50:28.677+0000] {logging_mixin.py:190} INFO - Dag name:ecommerce_data_pipeline and current dag run status:running
[2025-03-30T01:50:28.678+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-30T01:50:28.680+0000] {Gestioncommands.py:40} INFO - Fetching data from API...
[2025-03-30T01:50:30.428+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/urllib3/connectionpool.py:1097: InsecureRequestWarning: Unverified HTTPS request is being made to host 'my.api.mockaroo.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings
  warnings.warn(

[2025-03-30T01:50:31.262+0000] {Gestioncommands.py:45} INFO - Data fetched successfully
[2025-03-30T01:50:31.265+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-30T01:50:31.279+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-30T01:50:31.280+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=ecommerce_data_pipeline, task_id=fetch_data, run_id=manual__2025-03-30T01:50:25.698169+00:00, execution_date=20250330T015025, start_date=20250330T015028, end_date=20250330T015031
[2025-03-30T01:50:31.313+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-03-30T01:50:31.315+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-03-30T01:50:31.316+0000] {logging_mixin.py:190} INFO - Dag name:ecommerce_data_pipeline queued_at:2025-03-30 01:50:25.718978+00:00
[2025-03-30T01:50:31.317+0000] {logging_mixin.py:190} INFO - Task hostname:778b271a0247 operator:PythonOperator
[2025-03-30T01:50:31.370+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-30T01:50:31.416+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-30T01:50:31.421+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
